client/state/data-layer/wpcom/sites/stats/google-my-business/index.js

```diff
@@ -64,3 +64,5 @@ registerHandlers( 'state/data-layer/wpcom/sites/stats/google-my-business/index.j
 		} ),
 	],
 } );
+
+export default {};
```

Comment:
This comes from https://github.com/Automattic/wp-calypso/blob/bbad57e43a740e0d5ea37224b9e4114a2ed8beed/client/state/data-layer/wpcom/sites/stats/index.js, which imports the default export of this file.

Looks like #26989 introduced this pattern, and it is being used in many other files. But I don't know if there is a more modern way of doing that. 

@tyxla @flootr, could you confirm?

Reply:
Aside from the fact that we might be going away from the data layer soon, the idea here is that ideally, no `mergeHandlers()` should be necessary in the data layer. Any state or functionality that requires a portion of the data layer would simply import that part of the data layer, and that's all. So the `google-my-business` one would be imported only by code that actually needs it and uses it. #26989 was part of the transition to that, but we never finished this migration. So for now, to avoid further refactors that won't be related to this PR, this will do the job.